charts/camunda-platform/README.md

```diff
@@ -1326,16 +1326,20 @@ Please see the corresponding [release guide](../../docs/release.md) to find out
 | `webModeler.restapi.sidecars`                                          | can be used to attach extra containers to the restapi deployment                                                                                                             | `[]`                             |
 | `webModeler.restapi.initContainers`                                    | can be used to set up extra init containers for the application Pod                                                                                                          | `[]`                             |
-| `webModeler.restapi.externalDatabase`                                  | can be used to configure a connection to an external database. This will only be applied                                                                                     |                                  |
+| `webModeler.restapi.externalDatabase`                                  | can be used to configure a connection to an external database; will only be applied if the postgresql dependency chart is disabled (with `postgresql.enabled=false`)         |                                  |
 | `webModeler.restapi.externalDatabase.url`                              | defines the JDBC url of the database instance                                                                                                                                | `""`                             |
-| `webModeler.restapi.externalDatabase.user`                             |                                                                                                                                                                              | `""`                             |
-| `webModeler.restapi.externalDatabase.password`                         | defines the database user's password                                                                                                                                         | `""`                             |
+| `webModeler.restapi.externalDatabase.user`                             | defines the database user                                                                                                                                                    | `""`                             |
+| `webModeler.restapi.externalDatabase.password`                         | can be used to provide the database user's password; ignored if `webModeler.restapi.externalDatabase.existingSecret` is set                                                  | `""`                             |
+| `webModeler.restapi.externalDatabase.existingSecret`                   | can be used to provide the name of an existing secret resource containing the database password                                                                              | `nil`                            |
```

Comment:
Should we mention `existingSecret.name` here? Otherwise, users might think that they can provide the name of the secret directly in `existingSecret`.
I noticed though that we don't mention it for the other places where this syntax is supported either.

Reply:
I think that's a tough one. this README.md is programmatically generated, so any update to it will be overwritten any time someone would change a comment in values.yaml.  

No other components provide extra details around their existingSecret option, so I think this is fine.